### PR TITLE
Milestone 3 : Using Bootstrap (Newsweek)

### PR DESCRIPTION
**Pull request by ZeenatLawal**
## Newsweek clone milestone 3: Using Bootstrap

In this milestone we:  

- Added the last sections of the website and adapted for the 3 screen sizes
- Used bootstrap's default media query breaking points (Tablet >= 768px, Desktop >= 992px) 
- Used bootstrap classes for the styling

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1003,6 +1001,545 @@ <h4><a href="#" class="link-dark nav-link p-0 mt-2"> Hokkaido: Japan’s Norther
...
+                <header><h5 class="otext fw-bold">NEWSLETTER</h5></header>
+                <div class="d-md-flex flex-md-row mt-lg-3 letter">
+                    <div class="phone"></div>
+                    <div class="border border-1 mb-4 ">
+                     <form class=" my-3 mx-2 px-lg-5">
+                            <div class="text-center text-lg-start">
+                              <h5 class="fw-bolder text-uppercase mt-lg-5">The Starting 5</h5>
+                              <p class="sm-font1 mt-lg-3">See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                              <div class="emaildiv">
+                              <input type="text"  class="email2 form-text border-light border-1" placeholder="Email address">
```
Comment (sinansevgi) [inline index.html]:
```suggestion
                              <input type="email"  class="email2 form-text border-light border-1" placeholder="Email address">
```
- [ ] You should use the input type `email ` for the email input fields.

File: index.html [hunk 2]
```diff
@@ -1003,6 +1001,545 @@ <h4><a href="#" class="link-dark nav-link p-0 mt-2"> Hokkaido: Japan’s Norther
...
+                            <h5 class="fw-bold otext"><span class="first-header text-white">PERISCOPE</span> CULTURE</h5>
+                            <p><a href="#" class="text-dark nav-link p-0 mt-2 fw-bold">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></p>
+                        </div>
+                    </div>
+                    <div class="col-md-3 container-fluid px-3 py-3">
+                        <div class="mb-2">
+                            <a href="#"><img src="image/sin.jpg"  class="img-width" alt="..."></a>
+                        </div>
+                        <div>
+                            <h5 class="fw-bold otext"><span class="first-header text-white">DOWNTIME</span> CULTURE</h5>
```
Comment (sinansevgi) [inline index.html]: - [ ] Titles also should have links in the `in the magazine` section.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 
-----

Hello team,
Good going!. Your project is almost done. There are some small things you should fix before you continue with your journey :wave: 

**_Please check the issues I left as inline comments._**

-----

## "As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project."

-----
You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

File: index.html (hunk 1 above)
Comment (descholar-ceo) [reply index.html]: resolved :heavy_check_mark:

File: index.html (hunk 2 above)
Comment (descholar-ceo) [reply index.html]: resolved :heavy_check_mark:

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @ZeenatLawal and @safafa  , your project is approved :tada: :tada: :tada: 
You have implemented all of the requested changes in the previous review, so I am marking your project as completed, but before merging all changes on the default branch, kindly fix your navbar on mobile screens the logo is too small hence it's making your clone messed on mobile screens,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**